admin/page/ajax-edit-berita.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+    include("../system/connection.php");
```

Comment:
Codacy found an issue: ["include" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)